tests/unit/test_tracking_cli.py

```diff
@@ -35,13 +35,13 @@ def disable_wandb(tmp_dir):
 
 @pytest.fixture
 def samples_dir():
-    return Path(__file__).parent / "data"
+    return Path(__file__).resolve().parents[1] / "data"
 
 
 @patch(
     "translations_parser.cli.taskcluster.get_args",
     return_value=argparse.Namespace(
-        input_file=Path(__file__).parent / "data" / "taskcluster.log",
+        input_file=Path(__file__).resolve().parents[1] / "data" / "taskcluster.log",
```

Comment:
To expand on my earlier comments around pathlib, I find this style to be really confusing where you are indexing into the array of parents.

I would prefer something simpler like:

`Path(__file__).parent / "../data/taskcluster.log"`